scripts/kata-install/osc-kata-install.sh

```
@@ -154,22 +153,48 @@ install_kata() {
+		    tar cf - $(rpm -qpl $rpm_path | grep "^/etc/") |
+			chroot /host tar xf -
+		done
+
+		# rpm-ostree --apply-live does not run RPM scriptlets
+		# Run the scriptlets manually in host context
+		# Normally this includes:
+		# 1. postinstall: Building host-kernel derived kata VM image
+		# 2. posttrans:	  Running the SELinux steps
+		for phase in postinstall posttrans; do
+		    for rpm_path in "${install_rpms[@]}"; do
+			scriptlet="$(rpm -qp --scripts $rpm_path | awk '/^'$phase' scriptlet/{found=1;next} /^[a-z]+ scriptlet/{found=0} found')"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)